requirements.txt

```diff
@@ -1,10 +1,14 @@
-pyyaml>=5.3.1
-numpy>=1.18.1
-numba==0.48.0  # Fix to librosa not respecting something deprecated
-librosa==0.7.2
 click>=7.1.1
+cytoolz>=0.10.1
+lhotse~=0.1
+librosa==0.7.2
 lilcom>=1.1.0
+matplotlib~=3.2.2
```

Comment:
The new dependencies added here shouldn't be necessary - you can do `pip install -e .[dev]` to install both lhotse and all of its dependencies. Pip supports installing different configurations, e.g. `pip install lhotse` or `pip install lhotse[dev]` (the dot `.` in the previous cmd corresponds to `lhotse`, it just tells pip to install from local sources). I recently ordered the dependencies to clearly separate what is needed by the end users and what is needed by the devs. If you install the `dev` version, nothing will be missing.